cmd/controller/app/options/options.go

```diff
@@ -150,6 +153,7 @@ const (
 	defaultMaxConcurrentChallenges   = 60
 
 	defaultPrometheusMetricsServerAddress = "0.0.0.0:9402"
+	defaultHealthzServerAddress           = "0.0.0.0:8081"
```

Comment:
Port 8081 is used by kubebuilder for healthz endpoints, so I copied that:
 * https://github.com/kubernetes-sigs/kubebuilder/blob/0fa57405d4a892efceec3c5a902f634277e30732/pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/manager/config.go#L132

Reply:
Change of plan. I used the Kubernetes controller-manager default instead.

https://github.com/kubernetes/kubernetes/blob/806b30170c61a38fedd54cc9ede4cd6275a1ad3b/pkg/cluster/ports/ports.go#L44-L46

Reply:
Another change of plan. I now use 9403 (metrics-server-port + 1).

Although I wonder if the  reasons for using port >=10250 is the same as @maelvls wrote in [the GKE Private Cluster: webhook troubleshooting guide](https://cert-manager.io/docs/troubleshooting/webhook/#gke-private-cluster):
> For context, unlike public GKE clusters where the control plane can freely talk to pods over any TCP port, the control plane in private GKE clusters can only talk to the pods in worker nodes over TCP port 10250 and 443. These two open ports refer to the containerPort inside the pod, not the port called port in the Service resource.
-- https://cert-manager.io/docs/troubleshooting/webhook/#gke-private-cluster